src/components/layout/nav/search/Search.scss

```diff
@@ -0,0 +1,211 @@
+  margin: auto;
+  height: 100px;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 0;
+
+  span {
+    position: absolute;
+    padding: 5px 15px;
+    background-color: #e2909066;
+    border-radius: 5px;
+    bottom: 5px;
+    max-width: 400px;
+    color: #464444;
+    font-size: 1.4vw;
+  }
+
+  .searchBar {
+    height: 55%;
+    background-color: #ebebeb;
+    border: 1px solid #d7d7d7;
+    color: #333;
+    position: relative;
+    align-items: center;
+    display: flex;
+    padding-left: 0;
+    padding-right: 0;
+    border-radius: 0.3rem;
+    transition: all 0.3s ease-in-out;
+
+    .searchButtonWrapper,
+    .searchInputWrapper {
+      height: 100%;
+      position: relative;
+      display: inline-block;
+      padding: 0;
+    }
+
+    .searchInput {
+      width: 100%;
+      height: 100%;
+      font-size: 16px;
+      background-color: #ebebeb;
+      transition: background-color 0.3s ease-in-out;
+      border-top-right-radius: 0 !important;
+      border-bottom-right-radius: 0 !important;
+
+      &:hover {
+        background-color: white;
+      }
+    }
+
+    button {
+      padding: 0;
+      height: 100%;
+      width: 100%;
+      background-size: 150%;
+      background-color: #8bc6ec;
+      background-image: linear-gradient(135deg, #8bc6ec 0%, #9599e2 100%);
+    }
+  }
+}
+
+.searchPreview {
+  background-color: #ffffff;
+  position: absolute;
+  display: flex;
+  top: 75%;
+  left: 50%;
+  transform: translateX(-50%);
+  padding-left: 0;
+  padding-right: 0;
+  border: 1px solid #d7d7d7;
+  border-bottom-right-radius: 0.3rem;
+  border-bottom-left-radius: 0.3rem;
+  border-top: none;
+  z-index: 1;
+  justify-content: center;
+  align-items: center;
+  padding-top: 20px;
+  padding-bottom: 20px;
+
+  img {
+    display: block;
+    margin: auto;
+    max-width: 70%;
+  }
+
+  .card {
+    padding-top: 10px;
+    padding-bottom: 10px;
+    background-color: #ffffff;
+    box-shadow: 1px 3px 20px rgba(0, 0, 0, 0.25),
+      -6px -6px 20px rgba(255, 255, 255, 0.25);
+    font-size: 16px;
+    margin-right: 20px;
+
+    &:hover {
+      cursor: pointer;
+    }
+
+    .title {
+      font-size: 1em;
+    }
+
+    .brand {
+      display: inline-block;
+      width: 70%;
+      opacity: 0.5;
+      font-size: 0.9em;
+    }
+
+    .price {
+      display: inline-block;
+      opacity: 0.5;
+      width: 30%;
+      text-align: end;
+      font-size: 0.9em;
+      position: absolute;
+      bottom: 10px;
+      right: 10px;
+    }
+  }
+}
+
+$breakPointPreviewBreaks: 1200px;
+$breakPointCardBreaks: 1134px;
+$breakPointHamburgerSwallowSearch: 769px;
+$breakPointOfSmallDevices: 620px;
+$breakPointOfExtraSmall: 444px;
+
+@media (max-width: breakPointPreviewBreaks) {
+  .searchPreview {
+    margin-top: 8px;
+  }
+}
+
+@media (max-width: $breakPointCardBreaks) {
+  .searchPreview {
+    .card {
+      font-size: 90%;
```

Comment:
for example: what's wrong here with simply using normal font-size for card breaks breakpoint? in general all these extra rules actually make your code more complicated and hard to maintain, i would recommend keeping it simpler

Reply:
I thought it is better to see where the `card` is located in but I can fix it of course.